lib/response.js

```diff
@@ -382,7 +384,11 @@ function noteActions (req, res, next) {
         actionInfo(req, res, note)
         break
       case 'pdf':
-        actionPDF(req, res, note)
+        if (config.allowPDFExport) {
+          actionPDF(req, res, note)
+        } else {
+          response.errorForbidden(res)
```

Comment:
We should also add `logger.error('PDF export failed: Disabled by config. Set "allowPDFExport: true" to enable. Check the documentation for details')` so no one is confused and we provide a helpful error message.

Reply:
@SISheogorath  Thanks for the suggestion. Added the error message